adapters/repos/db/shard_backup.go

```diff
@@ -138,8 +143,8 @@ func (s *Shard) readBackupMetadata(d *backup.ShardDescriptor) (err error) {
 	return nil
 }
 
-func (s *Shard) nodeName() string {
-	node, _ := s.index.getSchema.ShardOwner(
+func (s *Shard) nodeName() (string, error) {
```

Comment:
**suggestion:** wdyt if we call it `ShardOwner()` to be consistent with what it's calling ?